<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlbumWidget</class>
 <widget class="QWidget" name="AlbumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="albumName">
        <property name="font">
         <font>
          <family>Impact</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addPictureButton">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/photo-add.png</normaloff>:/icons/photo-add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editAlbumButton">
        <property name="text">
         <string>Modify</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/album-edit.png</normaloff>:/icons/album-edit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeAlbumButton">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/album-delete.png</normaloff>:/icons/album-delete.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="thumbnailsListView"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
